compiler/plc_diagnostics/src/diagnostics/error_codes/E099.md

```diff
@@ -0,0 +1,8 @@
+# Invalid `REFERENCE TO` declaration
+
+`REFERENCE TO` variable declarations are considered valid if the referenced type is not of the following form
+* `foo : REFERENCE TO REFERENCE TO (* ... *)`
+* `foo : ARRAY[...] OF REFERENCE TO (* ... *)`
+* `foo : REF_TO REFERENCE TO (* ... *)`
+
+Furthermore `REFERENCE_TO` variables must not be initialized in their declaration, e.g. `foo : REFERENCE TO DINT := bar`.
```

Comment:
fyi, this is not entirely correct but will be addressed in the aliasing PR later on